### ISSUE TITLE
Log Data capture - stemming from CARLI data review

### ISSUE THREAD

**Issue by patrickzurek** [eXtensibleCatalog/xcmetadataservicestoolkit#638]
JIRA issue created by: rcook
Originally opened: 2011-06-21 01:38 PM

Issue body:
 (nt)

**Comment by patrickzurek** [comment]
JIRA Coment by user: rcook
JIRA Timestamp: 2011-06-21 01:40 PM

Comment body:

Some issues captured here are data problems in the original data that the library should be notified about in a log or something.    

We should make sure specs are clear and then assign to John Brand.

http://code.google.com/p/xcmetadataservicestoolkit/wiki/RecordCounts

**Comment by patrickzurek** [comment]
JIRA Coment by user: jbowen
JIRA Timestamp: 2011-09-02 04:47 PM

Comment body:

Think this one should be closed.  I've documented changes for Transformation that this data brought up.  I tried to look at the data again but a lot of time has elapsed and the original output data appears to be gone.  So, no way to notify UIC about individual record problems.  If they are still there in the next run of their data I'm sure they'll pop up again.
